packages/auth/src/AuthProvider.tsx

```diff
@@ -16,7 +16,7 @@ export interface AuthContextInterface {
   /* Determining your current authentication state */
   loading: boolean
   isAuthenticated: boolean
-  authToken: string | null
+  authToken: string | null // @WARN! to be depracated
```

Comment:
I think if you use JSDocs then it'll actually warn the user in vscode:

```suggestion
 /**
  * @deprecated auth tokens are now refreshed when they expire, and this value will be removed from the context in the next few releases: https://github.com/redwoodjs/redwood/pull/1609
  */
  authToken: string | null // @WARN! to be depracated
```

```